Bugsnag/BSGOutOfMemoryWatchdog.m

```diff
@@ -274,17 +274,19 @@ - (NSDictionary *)readSentinelFile {
         bsg_log_err(@"Failed to read oom watchdog file: %@", error);
         return nil;
     }
-    NSDictionary *contents = [BSGJSONSerialization JSONObjectWithData:data options:0 error:&error];
+    NSMutableDictionary *contents = [[BSGJSONSerialization JSONObjectWithData:data options:0 error:&error] mutableCopy];
```

Comment:
```suggestion
    NSMutableDictionary *contents = [[BSGJSONSerialization JSONObjectWithData:data options:NSJSONReadingMutableContainers error:&error] mutableCopy];
```

Reply:
Actually, we shouldn't need the `mutableCopy` call now right?

Reply:
correct!